block-storage/src/main/scala/coop/rchain/blockstorage/ApprovedStore.scala

```diff
@@ -0,0 +1,27 @@
+package coop.rchain.blockstorage
+
+import cats.effect.Sync
+import cats.syntax.all._
+import coop.rchain.blockstorage.dag.codecs.codecApprovedBlock
+import coop.rchain.casper.protocol.{ApprovedBlock, ApprovedBlockProto}
+import coop.rchain.shared.syntax._
+import coop.rchain.store.{KeyValueStoreManager, KeyValueTypedStore}
+import scodec.codecs._
+
+object approvedStore {
```

Comment:
Another nitpick.
`ApprovedStore` instead of `approvedStore`. Just some warning I saw in intellJ:)

Reply:
Yes, it looks strange but here is [reason](https://github.com/rchain/rchain/pull/3655#discussion_r848253129) of lower-case object name.

Reply:
@zsluedem This object definition has the purpose like a package which starting with lower case makes sense (and to avoid hiding alias and implicit instance helper).
https://docs.scala-lang.org/style/naming-conventions.html#objects

Reply:
Hm.... The link says `Object names are like class names (upper camel case).`

While you could `mimicking a package or function` with lower case. What a confusion!

Reply:
I just tested with upper case and it can work also (I think I misunderstood the error in syntax file when I was refactoring).
```scala
object BlockStore {
  type BlockStore[F[_]] = KeyValueTypedStore[F, BlockHash, BlockMessage]

  def apply[F[_]](implicit instance: BlockStore[F]): instance.type = instance
}
```
This changes a lot of files so better to do it in a separate PR to minimize conflicts.